### ISSUE TITLE
Adding more hints instead of overriding in settings file

### ISSUE THREAD

**Issue by Krzysztof-Cieslak** [fsprojects/FSharpLint#229]
Right now setting `Hints` entry in configuration file override default settings. In many cases I don't want to override hints, I just want to add few other entries. To do so I need to come here, copy all default hints, and put them in my custom config file.  Instead I'd like to do something like:
```
        <Hints>
            <Hints>
              <![CDATA[
                %INCLUDE_DEFAULT_HINTS%
                Option.get x ===> m"Consider using pattern matching - this function is partial and can throw exception"
                List.head x ===> m"Consider using `List.tryHead` and pattern matching - this function is partial and can throw exception"
                List.find f x ===> m"Consider using `List.tryFind` and pattern matching - this function is partial and can throw exception"
              ]]>
            </Hints>
            <Enabled>true</Enabled>
        </Hints>
```

**Comment by Krzysztof-Cieslak** [comment]
Fixed by #230

**Comment by duckmatt** [comment]
Thanks for the suggestion, this will now be the default behaviour.

Old behaviour of overwriting can be done via an update attribute on the Hints node:

        <Hints>
            <Hints Update="Overwrite">
              <![CDATA[
                Option.get x ===> m"Consider using pattern matching - this function is partial and can throw exception"
                List.head x ===> m"Consider using `List.tryHead` and pattern matching - this function is partial and can throw exception"
                List.find f x ===> m"Consider using `List.tryFind` and pattern matching - this function is partial and can throw exception"
              ]]>
            </Hints>
            <Enabled>true</Enabled>
        </Hints>
